# HG changeset patch
# User Rik <rik@octave.org>
# Date 1448818292 28800
#      Sun Nov 29 09:31:32 2015 -0800
# Node ID cd50ce72fa243b2e05b4c6a25f51d2efee744dc6
# Parent  5c5e085a1ae637137035c5f5f4dea585d6d0c15d
parser.tst: Add test for resolved bug #46534.

* parser.tst: Add test for resolved bug #46534.

diff --git a/test/parser.tst b/test/parser.tst
--- a/test/parser.tst
+++ b/test/parser.tst
@@ -311,8 +311,12 @@
 %! assert (a, 'ab');
 
 ## Check that a cell array containing function handles is parsed
 ## correctly with or without commas.
 %!test
 %! a = {1, @sin, 2, @cos};
 %! b = {1 @sin 2 @cos};
 %! assert (a, b)
+
+## Maybe unnecessary, but check that further changes to parser don't 
+## invalidate error handling (bug #46534).
+#!error <vertical dimensions mismatch \(1x2 vs 1x1\)> z = [1, 2; 3]
